src/test/java/de/tum/cit/aet/artemis/core/FileIntegrationTest.java

```diff
@@ -387,8 +405,14 @@ void testGetAttachmentUnitAttachmentFilenameSanitization() throws Exception {
         attachmentRepo.save(attachment);
         attachmentUnitRepo.save(attachmentUnit);
 
-        String unsanitizedFilename = unsanitizedName + ".pdf";
-        String url = "/api/core/files/attachments/attachment-unit/" + attachmentUnit.getId() + "/" + unsanitizedFilename;
+        String unsanitizedFilename = "AttachmentUnit_2025-05-10T12-10-34_" + unsanitizedName + ".pdf";
+        String url;
+        if (isTutor) {
+            url = "/api/core/files/attachments/attachment-unit/" + attachmentUnit.getId() + "/" + unsanitizedFilename;
+        }
+        else {
+            url = "/api/core/files/attachments/attachment-unit/" + attachmentUnit.getId() + "/student/" + unsanitizedFilename;
+        }
```

Comment:
I would argue that this is easier to read and maintain with a ternary operator
```suggestion
        String url = isTutor
            ? "/api/core/files/attachments/attachment-unit/" + attachmentUnit.getId() + "/" + unsanitizedFilename
            : "/api/core/files/attachments/attachment-unit/" + attachmentUnit.getId() + "/student/" + unsanitizedFilename;
```